src/main/scala/nl/knaw/dans/easy/stage/dataset/AMD.scala

```diff
@@ -22,23 +22,29 @@ import scala.xml.Elem
 object AMD {
   type AdministrativeMetadata = Elem
 
-  def apply(depositorId: String, submissionTimestamp: DateTime, draft: Boolean): AdministrativeMetadata = {
-
-    val lastState = {if (draft) "DRAFT" else "SUBMITTED"}
-
+  def apply(depositorId: String, submissionTimestamp: DateTime, state: String): AdministrativeMetadata = {
     <damd:administrative-md xmlns:damd="http://easy.dans.knaw.nl/easy/dataset-administrative-metadata/" version="0.1">
-      <datasetState>{lastState}</datasetState>
-      <previousState>DRAFT</previousState>
-      <lastStateChange>{submissionTimestamp}</lastStateChange>
+      <datasetState>{state}</datasetState>{
```

Comment:
Improved the similarity with the Web-UI somewhat. If the state is `DRAFT` in the Web-UI there is no previous state.